tests/vmi_lifecycle_test.go

```diff
@@ -77,7 +76,7 @@ import (
 	"kubevirt.io/kubevirt/tests/watcher"
 )
 
-var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level:component][sig-compute]VMIlifecycle", decorators.SigCompute, decorators.VMIlifecycle, func() {
```

Comment:
For the record: it is marked as `[arm64]` here, but the test actually skips arm64.
Great catch!